README.rst

```diff
@@ -1,81 +1,98 @@
-This package contains the Zephyr RTOS meta tool, 'west'.
+This is the Zephyr RTOS meta tool, ``west``.
 
-WARNING
--------
+Installation
+------------
 
-DO NOT INSTALL WITH "python3 setup.py install".
+Install West's bootstrapper with pip::
 
-Please use pip to install in development mode as documented below.
+  pip3 install west
 
-Important Note
---------------
+Then install the rest of West and a Zephyr development environment in
+a directory of your choosing::
 
-West is distributed in two pieces:
+  mkdir zephyr-dir && cd zephyr-dir
+  west init
+  west fetch
 
-1. A bootstrap/wrapper script, which is distributed via PyPI.
-2. The "main" west package and entry points, which are fetched by the
-   bootstrap script.
+What just happened:
 
-This somewhat unusual arrangement is because:
+- ``west init`` runs the bootstrapper, which clones the West source
+  repository and a *West manifest* repository. The manifest contains a
+  YAML description of the Zephyr installation, including Git
+  repositories and other metadata.
 
-- One of West's jobs is to manage interaction with Zephyr's multiple
-  Git repositories, including its own.
-- West is in its experimental stages and is moving quickly, meaning
-  users need to stay on HEAD.
+- ``west fetch`` clones the repositories in the manifest, creating
```

Comment:
maybe comment here that `west fetch` is run by the freshly cloned `west` code

Reply:
I'm not sure what this means ... ``west fetch`` isn't run by the code, not sure how code can run something. Sorry to be dense, could you please clarify?

Reply:
Just after installing `west` from PyPi with `pip`, you can run `west fetch`. The Python script that actually executes all the required `git fetch` operations is the one in the `west/` folder, not the one installed by pip, isn't it? This is no big deal, just a small clarification that might or might not be applicable.

Reply:
Nice, thank you